docs/GetStarted/getting_started_python.md

```diff
@@ -0,0 +1,61 @@
+# Getting Started with Python
+
+IREE has Python bindings geared towards lower level compiler interop that are
+not intended to be a public API, and integration with Python frontends such as
+TensorFlow.
+
+We do not yet provide a pip package for easy installation, so to use IREE's
+Python bindings you must build from source.
+
+## Prerequisites
+
+You should already have IREE cloned and building on your machine. See the other
+[getting started guides](.) for instructions.
+
+> Note:<br>
+> &nbsp;&nbsp;&nbsp;&nbsp;Support is best with Bazel.
+> For CMake (excluding TensorFlow), set the `IREE_BUILD_PYTHON_BINDINGS` option.
+
+## Python Setup
+
+Install a recent version of [Python 3](https://www.python.org/downloads/) and
+[pip](https://pip.pypa.io/en/stable/installing/), if needed.
+
+Install packages:
+
+```shell
+$ pip install numpy
+
+# If using the TensorFlow integration
+$ pip install tf-nightly
+```
+
+## Running Python Tests
+
+To run tests for core Python bindings:
+
+```shell
+$ bazel test bindings/python/...
+```
+
+To run tests for the TensorFlow integration, which include end-to-end backend
+comparison tests:
+
+```shell
+# Exclude tests that are skipped in the GitHub Actions ("ga") CI
+$ bazel test \
+  --build_tag_filters="noga" \
+  --test_tag_filters="noga" \
+  --define=iree_tensorflow=true \
```

Comment:
I'm not sure we need this anymore. Also not 100% sure we don't.

Reply:
Still used here: https://github.com/google/iree/blob/b2c034f78f0eb3f85cb50ff48ba594c4aa12b5aa/colab/BUILD.bazel#L27-L33